### PR TITLE
[WIP] Add logs to debug testHighConcurrencyModificationOfResourceAcls test case

### PR DESCRIPTION
**Pull request by omkreddy**
Not able to reproduce locally. added few logs to check on jenkins run
looks like some synchronization issue

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala [hunk 1]
```diff
@@ -244,12 +244,14 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
...
     * @return boolean indicating if a change was made
     */
-  private def updateResourceAcls(resource: Resource)(getNewAcls: Set[Acl] => Set[Acl]): Boolean = {
+  private def updateResourceAcls(resource: Resource, acls: Set[Acl], add: Boolean)(getNewAcls: Set[Acl] => Set[Acl]): Boolean = {
     var currentVersionedAcls =
       if (aclCache.contains(resource))
         getAclsFromCache(resource)
       else
         getAclsFromZk(resource)
+    error(s"updateResourceAcls acls: $acls add : $add , currentVersionedAcls = $currentVersionedAcls")
```
Comment (tedyu) [inline core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala]: In a normal run where test passes, I see this.
Consider changing to debug or info.

**Comment by omkreddy** [comment]
retest this please

File: core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala (hunk 1 above)
Comment (omkreddy) [reply core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala]: just added for testing with jenkins job. Added ERROR log level to reduce amount of logs

**Comment by omkreddy** [comment]
retest this please

**Comment by omkreddy** [comment]
retest this please

**Comment by omkreddy** [comment]
retest this please

**Comment by omkreddy** [comment]
retest this please

**Pull request closed without merging**